### ISSUE TITLE
bad result in new dataset without fine-tune

### ISSUE THREAD

**Issue by lixiangweiabc** [ShunyuanZheng/GPS-Gaussian#13]
I directly used your published model weights（GPS-GS_stage2_final.pth） on a new dataset and I got a bad result.
The cmd is :
`python test_view_interp.py --test_data_root 'data/humanNeRF-data'  --ckpt_path 'data/GPS-GS_stage2_final.pth'  --novel_view_nums 9`.
This is the result:
![0029_novel06](https://github.com/ShunyuanZheng/GPS-Gaussian/assets/50644391/10a8e8cc-a826-4058-b354-c09507f3813e)
![0039_novel03](https://github.com/ShunyuanZheng/GPS-Gaussian/assets/50644391/b4c76bda-7497-4b73-8800-bef81b00e2b3)

This is the source views:
<img width="665" alt="image" src="https://github.com/ShunyuanZheng/GPS-Gaussian/assets/50644391/f8c7e96a-898c-4f60-b9ad-206c3ccc7a73">
Does it need fine tune in a new dataset? Or may be the source views are too  sparse.

**Comment by ShunyuanZheng** [comment]
Hi,

The provided pre-train model is trained on the synthetic data under a 16-camera and 2m-radius setting. The model cannot be directly generalized to a test set with significantly different scene settings. If tested on a new dataset, I recommend generating the training data under the targeted scene and training the two stages from scratch. Also, I think the 6-camera setting is somewhat too sparse. Nonetheless, you can try to train GPS-Gaussian with THuman2.0 data and test on your custom data. I think the result will be much better than those displayed here.
